### PR TITLE
UI for list screen

### PR DESCRIPTION
**Pull request by DiegoQuiros94**
Trello card: https://trello.com/c/SOcHfMPy/9-lista-de-libros

Screenshots:

iPhone 11:

image

NexusS:

image

iPhone SE:

pending...

### CODE REVIEW

File: src/app/components/BookDetail/styles.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { StyleSheet } from 'react-native';
 
-import IOSSHADOW from '@app/constants/commonStyles';
-import { TUNDORA, WHITE } from '@app/constants/colors';
+import IOSSHADOW from '@constants/commonStyles';
+import { TUNDORA, WHITE } from '@constants/colors';
```
Comment (sfernandez11) [inline src/app/components/BookDetail/styles.ts]: 👍

File: src/Interfaces/Book.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export interface Book {
```
Comment (sfernandez11) [inline src/Interfaces/Book.ts]: Check folder and file name: Typo: `Interfaces/Book.ts` => `interfaces/book.ts`

File: src/app/components/BookDetail/index.tsx [hunk 3]
```diff
@@ -3,26 +3,31 @@ import { Text, View, Image } from 'react-native';
 
 import styles from './styles';
 
-interface Book {
-  title: string;
-  subtitle: string;
-  url: string;
+import { Book } from 'src/Interfaces/Book';
```
Comment (sfernandez11) [inline src/app/components/BookDetail/index.tsx]: `import { Book } from '@interfaces/book';`

File: src/app/index.tsx [hunk 4]
```diff
@@ -1,29 +1,16 @@
 import React from 'react';
 import { SafeAreaView, StatusBar } from 'react-native';
 
-import BookDetail from './components/BookDetail';
-import { WHITE } from './constants/colors';
+import { WHITE } from '../constants/colors';
+
+import HomeScreen from './screens/HomeScreen';
```
Comment (sfernandez11) [inline src/app/index.tsx]:
```
import { WHITE } from '@constants/colors';
import HomeScreen from '@screens/HomeScreen';
```

File: src/app/screens/HomeScreen/constants.ts [hunk 5]
```diff
@@ -0,0 +1,86 @@
+import { Book } from 'src/Interfaces/Book';
```
Comment (sfernandez11) [inline src/app/screens/HomeScreen/constants.ts]: `import { Book } from '@interfaces/book';`

File: src/app/screens/HomeScreen/index.tsx [hunk 6]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { FlatList, ListRenderItem } from 'react-native';
+
+import styles from './styles';
+import { DATA } from './constants';
+
+import BookDetail from '@app/components/BookDetail';
+import { Book } from 'src/Interfaces/Book';
```
Comment (sfernandez11) [inline src/app/screens/HomeScreen/index.tsx]: Move these two imports below global imports:
```
import React from 'react';
import { FlatList, ListRenderItem } from 'react-native';
import BookDetail from '@components/BookDetail';
import { Book } from '@interfaces/book';

import styles from './styles';
import { DATA } from './constants';
```

**Review by sfernandez11 — CHANGES_REQUESTED** [review]
Great work! Some comments...

File: src/app/index.tsx (hunk 4 above)
Comment (DiegoQuiros94) [reply src/app/index.tsx]: why is linter not letting me put them together if i don't use the path shortcuts?

File: src/app/screens/HomeScreen/index.tsx (hunk 6 above)
Comment (DiegoQuiros94) [reply src/app/screens/HomeScreen/index.tsx]: I tried it but eventually the prettier ends up setting it like this

**Review by sfernandez11 — APPROVED** [review]
🥇

File: src/app/screens/HomeScreen/index.tsx (hunk 6 above)
Comment (sfernandez11) [reply src/app/screens/HomeScreen/index.tsx]: My bad, bad linter config

File: src/app/index.tsx (hunk 4 above)
Comment (sfernandez11) [reply src/app/index.tsx]: Bad linter config, working correctly now 😄

**Pull request merged**